tests/before_script.sh

```diff
@@ -29,7 +29,7 @@ mkdir plugins/fusioninventory && git clone --depth=1 $FI_SOURCE -b $FI_BRANCH pl
 IFS=/ read -a repo <<< $TRAVIS_REPO_SLUG
 mv ../${repo[1]} plugins/flyvemdm
 cd plugins/fusioninventory
-patch -p1 < ../flyvemdm/tests/patches/fi-fix-obsolete-query.patch
+if [[ $FI_BRANCH == "glpi9.2+1.0" ]] ; then patch -p1 < ../flyvemdm/tests/patches/fi-fix-obsolete-query.patch; fi
```

Comment:
FI has a release 9.2+2.0. It is worth to run unit tests with this version and see if the patch may be dropped.

Reply:
For that version of FI the query sentence hasn't been updated so the patch is working but for the master branch the code was updated and that patch is useless.

Reply:
OK